linera-base/src/crypto/secp256k1/mod.rs

```diff
@@ -31,10 +32,10 @@ use crate::doc_scalar;
 const SECP256K1_SCHEME_LABEL: &str = "secp256k1";
 
 /// Length of secp256k1 compressed public key.
-const SECP256K1_PUBLIC_KEY_SIZE: usize = 33;
+pub const SECP256K1_PUBLIC_KEY_SIZE: usize = 33;
 
 /// Length of secp256k1 signature.
-const SECP256K1_SIGNATURE_SIZE: usize = 64;
+pub const SECP256K1_SIGNATURE_SIZE: usize = 64;
```

Comment:
Can we just use `mem::size_of` instead? (Because in this case it _is_ about the in-memory size.)